engine/src/flutter/shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -506,11 +518,16 @@ private void ensureValidRequest(
     }
   }
 
+  /** Returns the platform views channel. */
```

Comment:
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-useless-documentation